examples/cassandra/README.md

```diff
@@ -305,6 +305,97 @@ UN  10.244.0.5  74.09 KB   256     49.7%             86feda0f-f070-4a5b-bda1-2ee
 UN  10.244.3.3  51.28 KB   256     51.0%             dafe3154-1d67-42e1-ac1d-78e7e80dce2b  rack1
 ```
 
+### Using a DaemonSet
+
+In Kubernetes a _[Daemon Set](../../docs/admin/daemons.md)_ can distribute pods onto Kubernetes nodes, one-to-one.  Like a _ReplicationController_ it has a selector query which identifies the members of it's set.  Unlike a _ReplicationController_ it has a node selector to limit which nodes are scheduled with the templated pods, and replicates not based on a set target number of pods, but rather assigns a single pod to each targeted node.
+
+An example use case: when deploying to the cloud, the expectation is that instances are ephemeral and might die at any time. Cassandra is built to replicate data across the cluster to facilitate data redundancy, so that in the case that an instance dies, the data stored on the instance does not, and the cluster can react by re-replicating the data to other running nodes.
```

Comment:
@erictune It's not just about enforcing only one pod per node. It's about enforcing one node per pod, no more, no less. So, to achieve this by setting nodePort, I'd have to constantly manually update the replicationController to have replicas == number of nodes. With a DaemonSet this is automatic. I don't see what the problem is with encouraging this - the purpose of DaemonSet is to place a single pod on each selected node, correct? So, if you want to have data replication over your entire cluster, and allocate Cassandra onto newly created instances automatically, is this not what DaemonSet is designed to do/